framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -330,6 +333,106 @@ public void processClassTree(ClassTree classTree) {
         super.visitClass(classTree, null);
     }
 
+    /**
+     * Check that the field invariant meets the following requirements:
+     *
+     * <p>1.) If the superclass of {@code classTree} has a field invariant, then the field invariant
```

Comment:
I numbered the list by hand so that I could refer to the numbers in the method body.